packages/controllers/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.9]
+### Added
+- reset plugins running state in PluginController `persist` ([#64](https://github.com/MetaMask/snaps-skunkworks/pull/64))
+
+### Changed
+- @metamask/controllers@15.0.0 ([#66](https://github.com/MetaMask/snaps-skunkworks/pull/66))
```

Comment:
```suggestion
- **BREAKING:** `@metamask/controllers@15.0.0` ([#66](https://github.com/MetaMask/snaps-skunkworks/pull/66))
  - This may cause incompatibilities with other versions of the `@metamask/controllers` package.
```